Post/tests.py

```diff
@@ -1,3 +1,10 @@
-# from django.test import TestCase
+from Post.models import post
 
-# Create your tests here.
+
+class Test_Post():
+
+    def test_post_creation(self):
+        post1 = post(nameOfPoster='Shoval', nameOfLocation='Dead Sea', photoURL='www.testPost.com', Description='test')
+
+        assert post1 is not None
+        assert isinstance(post1, post)
```

Comment:
I find this too basic.
1. Instantiation of a class cannot be None in any case, no need to check for it.
2. It is obvious that the object which was instantiated from the `post` class is of this class type.

Perhaps add some content to check. Like define a `__str__` on that model and return some text representation of its content. Then you can do something like `assert str(post1) == "your expected content"`.

If you prefer not to do all this, even an `assert True` is better IMO.

Reply:
I changed the test as you asked. 
Is it good now?